reliability-engineering/terraform/modules/concourse-worker-pool/route53.tf

```diff
@@ -1,11 +1,11 @@
 data "aws_route53_zone" "private_root" {
-  zone_id = "${var.private_root_zone_id}"
+  zone_id = var.private_root_zone_id
 }
 
 locals {
-  concourse_url = "${replace(
+  concourse_url = replace(
     "web.${data.aws_route53_zone.private_root.name}",
     "/[.]$/",
-    ""
-  )}"
+    "",
```

Comment:
is this ok?

Reply:
I think so? It's a bit weird because it's a tool-assisted refactor but I think this looks equivalent. It certainly doesn't cause any unexpected changes in the consumer of this module.